cypress/objects/timeLoginPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoginPage {
+    visit() {
+        cy.visit('/')
+    }
+
+    getURL() {
+        return cy.url().should('include', '/time-logging')
+    }
+
+    getPageTitle() {
+        return cy.get('.page__title').contains('Timesheets')
+    }
+
+    getCalendarVisable() {
+        return cy.get('.calendar').should('be.visible')
+    }
+
+    getCalendarToday() {
+        return cy.get('.calendar--today').contains(new Date().getDate())
+    }
+
+    getFileForm() {
```

Comment:
This function's name should be way more descriptive, something like "`getTimeEntryForm()`" (because that's what it is, right?). This way you'll know for sure what this function is meant to get :)